### PR TITLE
First half of Fellowships

### PR DESCRIPTION
**Pull request by gvn**

### CODE REVIEW

**Comment by gvn** [comment]
This includes the FAQ and Overview tabs.

File: app/react/pages/fellowships/fellowships.jsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+              <h3 className="m-b-3">Who can become a Fellow?</h3>
+
+              <div className="m-b-3">
+                <IconItem imgSrc="/assets/img/fellowships/icon-institution.svg" copy="Currently working at a research institution in the US, UK or Canada."></IconItem>
+                <IconItem imgSrc="/assets/img/fellowships/icon-research.svg" copy="Currently an early-career researcher (i.e, graduate students, post-docs, research scientists, lecturers)."></IconItem>
+                <IconItem imgSrc="/assets/img/fellowships/icon-life.svg" copy="Specializing in the life sciences or natural sciences."></IconItem>
+                <IconItem imgSrc="/assets/img/fellowships/icon-travel.svg" copy="Able to travel."></IconItem>
+                <IconItem imgSrc="/assets/img/fellowships/icon-approve.svg" copy="Supported by their supervisors. As fellows will be based at their home institutions and a letter of support from their supervisor is mandatory."></IconItem>
+                <IconItem imgSrc="/assets/img/fellowships/icon-open.svg" copy="Have experience participating in open communities."></IconItem>
```
Comment (alanmoo) [inline app/react/pages/fellowships/fellowships.jsx]: There's a text gremlin after the period in the copy prop

**Comment by alanmoo** [comment]
The tabs component is not mobile yet, correct?

**Comment by gvn** [comment]
Correct, I need to refactor the component.

**Comment by alanmoo** [comment]
Which ticket does this close?

**Comment by gvn** [comment]
It's roughly half of #121

**Pull request merged**